### DISCUSSION TITLE
Missing config file

### DISCUSSION THREAD

**Discussion by Gwemlin** [vmatare/thinkfan#218] [Support]
Running Ubuntu on a modded T430, installed thinkfan but the config file is missing from /etc. I was going to just write a config file elsewhere and have it use that but if the config file is missing, I'm worried something more important might be as well. Honestly not sure what to do here, Help?

**Comment by vmatare** [comment]
Check `/usr/share/doc/thinkfan`. That's where your package manager should install documentation (in theory). Be aware though that the Ubuntu folks are responsible for packaging and installing, so it's probably better to ask the Ubuntu community, or (if something is truly missing) open a ticket on the Ubuntu bug tracker.
